ui/test/nuxeo-slots.test.js

```diff
@@ -15,6 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 import { fixture, flush, html } from '@nuxeo/nuxeo-elements/test/test-helpers.js';
+import * as polymer from '@polymer/polymer';
```

Comment:
`import *` shouldn't we avoid this pattern?

Reply:
We should avoid it if possible. But in this case, we want to use `html` from `polymer` and we're already importing `html` in the line above, which is lit's `html` function. We have this situation on other tests and we've used:
```
import * as polymer from '@polymer/polymer';
```
So this is consistent with what we've been doing so far.

A possible alternative would be something like:
```
import { html as polymerHtml } from '@polymer/polymer';
```
But I'd say using a `polymerHtml` for templating is uglier than `polymer.html`. WDYT?